GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift

```diff
@@ -21,14 +20,19 @@ struct DogBreedsListView: View {
             } else if viewModel.isLoading {
                 ActivityIndicator(isAnimating: true)
             } else {
-                List(viewModel.dogBreeds) { dogBreed in
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text(dogBreed.name)
-                        Text("min life: \(dogBreed.minLife), max life: \(dogBreed.maxLife)")
+                NavigationView {
+                    List(viewModel.dogBreeds) { dogBreed in
+                        NavigationLink(destination: EmptyView()) {
```

Comment:
Since you have created the detail view already, just use `DogBreedDetailView` instead of `EmptyView`

Reply:
I haven't read your comments(Because of my bad network connection), so  I did it myself.